app/src/main/java/com/github/onlynotesswent/ui/common/GridComposables.kt

```diff
@@ -70,17 +67,23 @@ fun CustomSeparatedLazyGrid(
           contentPadding = PaddingValues(vertical = 20.dp),
           horizontalArrangement = Arrangement.spacedBy(4.dp),
           modifier = gridModifier) {
-            items(sortedFolders, key = { it.id }, span = { GridItemSpan(2) }) { folder ->
-              FolderItem(
-                  folder = folder,
-                  navigationActions = navigationActions,
-                  noteViewModel = noteViewModel,
-                  folderViewModel = folderViewModel) {
-                    folderViewModel.selectedParentFolderId(folder.parentFolderId)
-                    navigationActions.navigateTo(
-                        Screen.FOLDER_CONTENTS.replace(
-                            oldValue = "{folderId}", newValue = folder.id))
-                  }
+            // If the current user is not the owner of the folders (except for search screen),
+            // only display the notes
```

Comment:
It is not as typo, basically when searching for a folder of other users, it will only show the notes inside and not the subfolders. We might change this in a later PR treating visibility of notes and folders